gemini/use-cases/retail/product_image_background_generation.ipynb

```diff
@@ -0,0 +1,461 @@
+   "source": [
+    "PROJECT_ID = \"[your-project-id]\"  # @param {type:\"string\"}\n",
+    "LOCATION = \"us-central1\"  # @param {type:\"string\"}\n",
+    "\n",
+    "import vertexai\n",
+    "\n",
+    "vertexai.init(project=PROJECT_ID, location=LOCATION)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "jXHfaVS66_01"
+   },
+   "source": [
+    "### Import libraries\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "lslYAvw37JGQ",
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from vertexai.generative_models import GenerationConfig, GenerativeModel, Image, Part"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "BY1nfXrqRxVX"
+   },
+   "source": [
+    "### Load the Gemini 1.5 Pro model\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "2998506fe6d1",
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "model = GenerativeModel(\"gemini-1.5-pro-preview-0409\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "Sr2Y3lFwKW1M"
+   },
+   "source": [
+    "### Define helper function"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "r_38e5rRKB6s"
+   },
+   "outputs": [],
+   "source": [
+    "import typing\n",
+    "import IPython.display\n",
+    "from PIL import Image as PIL_Image\n",
+    "from PIL import ImageOps as PIL_ImageOps\n",
+    "\n",
+    "\n",
+    "def display_image(\n",
+    "    image: Image,\n",
+    "    max_width: int = 600,\n",
+    "    max_height: int = 350,\n",
+    ") -> None:\n",
+    "    pil_image = typing.cast(PIL_Image.Image, image._pil_image)\n",
+    "    if pil_image.mode != \"RGB\":\n",
+    "        # RGB is supported by all Jupyter environments (e.g. RGBA is not yet)\n",
+    "        pil_image = pil_image.convert(\"RGB\")\n",
+    "    image_width, image_height = pil_image.size\n",
+    "    if max_width < image_width or max_height < image_height:\n",
+    "        # Resize to display a smaller notebook image\n",
+    "        pil_image = PIL_ImageOps.contain(pil_image, (max_width, max_height))\n",
+    "    IPython.display.display(pil_image)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "XibcO-Ev1xWi"
+   },
+   "source": [
+    "### Download a sample product image from Google Cloud Storage\n",
+    "\n",
+    "Use the ```Image.load_from_file``` method to load a local file as the example product image.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "BXiqOd3NOeSs"
+   },
+   "outputs": [],
+   "source": [
+    "# Download an image from Google Cloud Storage\n",
+    "! gsutil cp \"gs://cloud-samples-data/generative-ai/image/suitcase.png\" ./image.png\n",
+    "\n",
+    "# Load from local file\n",
+    "product_image = Image.load_from_file(\"image.png\")\n",
```

Comment:
nit: You can use `Part.from_uri()` to load directly from the GCS Path without downloading locally.

Only issue is that it's more difficult to display the image.